docs/source/platform/operation-registry.md

```diff
@@ -389,17 +395,17 @@ To register operations under a specific variant/tag:
 const server = new ApolloServer({
   plugins: [
     require("apollo-server-plugin-operation-registry")({
-      schemaTag: "prod" // highlight-line
+      schemaTag: "prod", // highlight-line
 
       // suggested before enforcing the safelist
       debug: true,
-      dryRun: true,
+      dryRun: true
     })
   ]
 });
 ```
 
-> Note: in order to copy the operations from one tag to another, please contact the Apollo team through Intercom or your shared Slack channel
+> Note: If you want to copy all previously registered operations to a new variant/tag, please contact the Apollo team
```

Comment:
nit: I think this could be a little friendlier of an offer for help. Something like
> We know it can be challenging to communicate with all consumers of the graph to publish their operations to a new tag. Reach out to the Apollo team if you want to move to a tag-based registry, and we'll be happy to migrate all operations registered to the `current` tag to the tag of your choosing to make that process easier.

Reply:
❤️ me gusta!